src/com/dabomstew/pkrandom/newgui/NewRandomizerGUI.java

```diff
@@ -1297,17 +1388,23 @@ public void mouseClicked(MouseEvent e) {
         JOptionPane.showMessageDialog(frame, messages);
     }
 
+    private void batchModeSettingsDialog() {
+        BatchModeSettingsDialog dlg = new BatchModeSettingsDialog(frame, batchModeSettings);
+        batchModeSettings = dlg.getCurrentSettings();
+        attemptWriteConfig();
+    }
+
     // This is only intended to be used with the "Keep Game Loaded After Randomizing" setting; it assumes that
```

Comment:
It would probably be a good idea to update this comment to explain that it's also used with batch randomization.

Reply:
Good catch. I will update the comment.